cli/objects/jira_base.py

```diff
@@ -308,3 +308,11 @@ def _get_security_level_id(
             f"Security level {security_level} not found in {project_key}, no security level will be applied.",
         )
         return None
+
+    def get_issue_by_id_or_key(self, issue_id_or_key: str) -> Issue:
+        return self.connection.issue(id=issue_id_or_key)
+
+    def add_labels_to_issue(self, issue_id_or_key: str, labels: list[str]) -> Issue:
+        issue = self.get_issue_by_id_or_key(issue_id_or_key)
+        issue.update(fields={"labels": labels + issue.fields.labels})
```

Comment:
Just curious, do you know if you try adding a label that already exists on an issue if it will cause an error? If so, we may want to check that labels in the `labels` list do not already exist on the issue and if they do, do not add the ones that already exist.

I have no idea if this is the case, just a thought for an edge case.

Reply:
It does not throw an error, but I am glad you asked this, because it made me think of a similar edge case in which accidental overwrites of labels could occur if they are added by an external source during the method's execution. I just tested and  pushed a change that should prevent this.